backend/curation/api/curation-api.yml

```diff
@@ -578,6 +707,28 @@ components:
         type: string
       nullable: true
       example: ["patient", "seqBatch"]
+    census_info_dataset:
+      type: object
+      properties:
+        collection_id:
+          $ref: "#/components/schemas/collection_id"
+        collection_name:
+          $ref: "#/components/schemas/collection_name"
+        collection_doi:
+          $ref: "#/components/schemas/doi"
+        dataset_id:
+          $ref: "#/components/schemas/dataset_id"
+        dataset_title:
+          description: |
+            [title](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#title)
+          nullable: true
+          type: string
+        dataset_h5ad_path:
+          type: string
```

Comment:
"Cell Census" must not appear in the swagger. 

Cell Census is only one example of a bulk downloader. The solution needs to be general purpose.